tests/filestore-v2.7-stream-depth/suricata.yaml

```diff
@@ -6,11 +6,12 @@ outputs:
       enabled: yes
       types:
         - files
-  - stream:
```

Comment:
@jasonish should we error on suricata.yaml having an unknown outputs section such as `stream` ?